build/l10n.js

```diff
@@ -70,6 +70,8 @@
     if (!containsFetchableLocale) {
       document.documentElement.dataset.noCompleteBug = true;
     }
+
+    L10n.registerLocales.call(this);
```

Comment:
Alternatively, you should be able to declare `meta` as local to the `initResoures` function, since its whole logic is sync now. This would also let you avoid having to clear `meta`.